cli/lib/kontena/scripts/completer.rb

```diff
@@ -160,6 +188,8 @@ def subcommand_tree(cmd = nil, base = nil)
         completion.clear
         sub_commands = %w(list use user current remove rm config cfg login logout token join audit-log init-cloud)
         if words[1] && words[1] == 'use'
+          completion.push helper.master_names.reject { |n| n == helper.current_master_name }
```

Comment:
^("master use" and "grid use" completions should not include the current master/grid)